code/camera/camera.cpp

```diff
@@ -608,6 +606,7 @@ subtitle::subtitle(int in_x_pos, int in_y_pos, const char* in_text, const char*
 	else
 		gr_init_alphacolor(&text_color, 255, 255, 255, 255);
 	text_fontnum = in_text_fontnum;
+	line_height_factor = in_line_height_factor;
```

Comment:
I'm not sure why this is set here instead of in the constructor's initializer list, but I see this pattern is used throughout.

Reply:
yeah, I'm not sure either *shrug*